### PR TITLE
View directory

### PR DESCRIPTION
**Pull request by Seven-Y-Q-Guo**
1. Add 'view directory' button on receive-form which can be redirect to file list page.
2. Fix undefined on shared address: https://github.com/parvardegr/sharing/pull/36/files#r994685400
3. Fix 404 error when using share functionality: https://github.com/parvardegr/sharing/pull/36/files#r994686684

Result for 1:
Screen Shot 2022-10-17 at 9 01 54 AM

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ $ sharing /path/to/file-or-directory -U user -P password  # also works with --re
     const uploadAddress = options.ip ? `${config.ssl.protocol}://${options.ip}:${options.port}/receive`: `${config.ssl.protocol}://${utils.getNetworkAddress()}:${options.port}/receive`;
 
     const time = new Date().getTime();
-    const file = encodeURIComponent(fileName);
+    const file = fileName ? encodeURIComponent(fileName) : '';
```
Comment (Seven-Y-Q-Guo) [inline bin/index.js:127]: Fix undefined on shared address
Screen Shot 2022-10-13 at 10 03 05 PM

File: bin/app.js [hunk 2]
```diff
@@ -57,8 +56,8 @@ const start = ({ port, path, receive, onStart, postUploadRedirectUrl }) => {
             });
         });
     }
-
-    app.get('/share/*', (req, res) => {
+    
+    app.use('/share', (req, res) => {
```
Comment (Seven-Y-Q-Guo) [inline bin/app.js:60]: Fix 404 error when using share functionality
Screen Shot 2022-10-13 at 10 03 55 PM

**Comment by parvardegr** [comment]
Hi @Seven-Y-Q-Guo very well, can you change the view directory button to just a simple link. (<a> instead of a button)
after that you can merge it 🙏

cheers 🥂

**Comment by Seven-Y-Q-Guo** [comment]
> Hi @Seven-Y-Q-Guo very well, can you change the view directory button to just a simple link. ( instead of a button) after that you can merge it 🙏
> 
> cheers 🥂

done

**Pull request merged**